### PR TITLE
Create a call for reviewers pull request template

### PR DESCRIPTION
**Pull request by awvwgk**
Here is what it looks like:

image

### CODE REVIEW

**Review by jvdp1 — APPROVED** [review]
Thanks

**Review by gareth-nx — APPROVED** [review]
Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!--
+Thank you for contributing to stdlib.
+To help us get your pull request merged more quickly, you can help us with reviewing any of the already open pull requests.
```
Comment (milancurcic) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: I suggest more active wording

```suggestion
To help us get your pull request merged more quickly, please consider reviewing any of the already open pull requests.
```

**Review by milancurcic — APPROVED** [review]
Thank you!

**Pull request merged**